src/Common/src/System/LocalAppContext.cs

```diff
@@ -35,29 +36,12 @@ private static bool GetCachedSwitchValueInternal(string switchName, ref int swit
             return isSwitchEnabled;
         }
 
-        private static bool DisableCaching 
-        { 
-            get 
+        private static bool DisableCaching =>
+            LazyInitializer.EnsureInitialized(ref s_disableCaching, ref s_isDisableCachingInitialized, ref s_syncObject, () =>
```

Comment:
@stephentoub sorry I updated the comment. Kindly check once again. This time I'm trying to understand instance property...